test/domain/authentication/_fixtures/login.fixture.ts

```diff
@@ -0,0 +1,95 @@
+
+import database, { RecordData } from '../../../../src/integrations/database/module';
+
+import { TooManySimilarNickNames } from '../../../../src/domain/creator/errors';
+import login from '../../../../src/domain/authentication/login';
+import { Identity } from '../../../../src/integrations/authentication/module';
```

Comment:
Integration module imports are commonly placed at the top of the file (like the database).